server/vcs/markdown/template_resolver.go

```diff
@@ -169,6 +174,18 @@ func (t *TemplateResolver) ResolveProject(result command.ProjectResult, baseRepo
 
 }
 
+func (t *TemplateResolver) filterOutput(s string) string {
+	var filteredLines []string
+	logLines := strings.Split(s, "\n")
+	for _, line := range logLines {
+		if t.LogFilter.ShouldFilterLine(line) {
+			continue
+		}
+		filteredLines = append(filteredLines, line)
+	}
+	return strings.Join(filteredLines, "\n")
```

Comment:
Nice! I like this spot!

I probably personally would have preferred something a bit more structured and reusable for the ansi stuff as well (and anything else we might want in the future).  You don't have to change it if its too much work but here's an example of something that could work.

```go

type StringFilter interface {
    Filter(s string) string
}

type FilterDecider interface {
    ShouldFilter(s string) bool
}

type RegexLineFilter struct {}

func (f RegexLineFilter) Filter (output string) string {
    for line := range strings.split("\n")  {
          // call ShouldFilterLine and append to an array
    }
}

func (f RegexLineFilter) ShouldFilter(s string) bool {
   // do the regex logic.
}

type AnsiFilter struct {}

func (a AnsiFilter) Filter(output string) string {
    // run the ansi stripper
}

// inject this in template resolver
type VCSTemplateFilters struct {
    Filters []StringFilter
}

func (f *TerraformLogFilter) Filter(output string) string {

    for filter := range f.Filters {
         output = filter.Filter(output)
    }

    return output
}

// inject this into project command output handler
var deciders []FilterDeciders []FilterDecider{ // regex line filter would go here }

```